wd_helper.au3

```diff
@@ -1025,6 +1027,47 @@ Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand)
 					_ArrayAdd($aOut, $sText, 0, Default, @CRLF, 1)
 					$vResult = $aOut
 				EndIf
+			Case 'options2'
+				; Retrieve array containing value / label attributes from the Select element's options - using faster way - RegExp
+				
+				; get outerHTML of Select Element
+				Local $sHTML = _WD_ElementAction($sSession, $sSelectElement, "Property", "outerHTML")
+				If @error Then
+					$iErr = @error
+				Else
+					; start checking if any is selected
+					Local $aOptions_outer = StringRegExp($sHTML, '(?is)(<option value="(.*?)"(.*?)>(.*?)<\/option>)', $STR_REGEXPARRAYGLOBALFULLMATCH)
+					If @error Then
+						$iErr = $_WD_ERROR_HtmlProcessing
+					Else
+						Local $aTemp
+						
+						; check which element is selected
+						For $iCheck = 0 To UBound($aOptions_outer) - 1
+							$aTemp = $aOptions_outer[$iCheck]
+							If StringInStr($aTemp[3], 'selected="selected"') Then
+								$iSelectedOption = $iCheck
+								ExitLoop
+							EndIf
```

Comment:
I don't believe this works the way you intended. AFAICS, `'selected="selected"'` only establishes the initially selected option and doesn't change when a new item is selected.

Reply:
This is no longer needed.